### DISCUSSION TITLE
Should we use GitHub Discussions feature?

### DISCUSSION THREAD

**Discussion by bourque** [spacetelescope/jwql#787] [General]
Discuss.

**Comment by bhilbert4** [comment]
Does this get us anything that the current slack channel doesn't?

**Reply by bourque** [reply]
Some things that come to mind are:
- It is more easy to link to existing GitHub issues/PRs
- Issues and PRs can reference discussions
- This has some sort of built-in voting feature (though this could be achieved in Slack with emojis)
- Ability to label and categorize discussions

That being said, i'm not yet sure what we would use this for.  Perhaps our own mini version of a helpdesk?

**Comment by bourque** [comment]
- @mfixstsci suggests this could be used as an 'pre-issue'/'issue staging area'
